Dashboard/app/js/lib/views/data/question-answer-view.js

```diff
@@ -74,6 +78,46 @@ FLOW.QuestionAnswerView = Ember.View.extend({
 
   numberValue: null,
 
+  cascadeValue: function(key, value, previousValue){
+    var c = this.content;
+    // setter
+    if (arguments.length > 1) {
+      // split according to pipes
+      var cascadeNames = value.split("|");
+      var cascadeResponse = [];
+      var obj = null;
+      cascadeNames.forEach(function(item){
+        if (item.trim().length > 0) {
+          obj = {};
+          obj.name = item.trim();
+          cascadeResponse.push(obj);
+        }
+      });
+
+      c.set('value', JSON.stringify(cascadeResponse));
+    }
+
+    // getter
+    var cascadeString = "", cascadeJson;
+    if (c && c.get('value')) {
+      if (c.get('value').indexOf("|") > -1) {
+        cascadeString += c.get('value');
+      } else {
+        cascadeJson = JSON.parse(c.get('value'));
+        for (var i = 0; i < cascadeJson.length; i++) {
+          cascadeString +=  cascadeJson[i].name + "|";
+        }
+
+        // strip last pipe
+        if (cascadeString.charAt(cascadeString.length - 1) === "|") {
+          cascadeString = cascadeString.substr(0, cascadeString.length - 1);
+        }
```

Comment:
I wonder if it would be clearer to use `join` here. Something like
```
cascadeString = cascadeJson.map(function(c) {
  return c.name;
}).join("|");
```
which takes care of not appending a trailing pipe.